### ISSUE TITLE
Broken in Sequoia

### ISSUE THREAD

**Issue by estiens** [airy10/FindMyDevices#1]
Just adding an issue here due to Sequoia updates to how it enctrypts stuff, please comment if someone turns up with a fix! Thanks

**Comment by airy10** [comment]
Right. Sequoia changed the access right to the keychain element containing the password needed to decrypt the files. It's now only readable by Apple FindMy processes

**Comment by akinomeroglu** [comment]
@airy10 hi, do you see any other way to add a regular user to this process user or group?

Thanks

**Comment by airy10** [comment]
I don't think that you can
